src/screens/PushTx.js

```diff
@@ -57,24 +69,89 @@ function PushTx() {
-      />
-      {canForce ? renderForceCheckbox() : null}
-      {success ? (
-        <span className="text-success">Transaction pushed to the network with success!</span>
-      ) : null}
-      {errorMessage ? <span className="text-danger">{errorMessage}</span> : null}
-    </div>
-  );
+  const renderUi = () => {
+    return (
+      <div className="content-wrapper">
+        <TxTextInput
+          ref={textRef}
+          buttonClicked={buttonClicked}
+          action="Push tx"
+          onChange={handleChangeData}
+          otherAction="decode"
+          link="/decode-tx/"
+          helpText="Write your transaction in hex value and click the button to send it to the network. (We do not push blocks to the network, only transactions)"
+        />
+        {canForce ? renderForceCheckbox() : null}
+        {success ? (
+          <span className="text-success">Transaction pushed to the network with success!</span>
+        ) : null}
+        {errorMessage ? <span className="text-danger">{errorMessage}</span> : null}
+      </div>
+    );
+  };
+
+  const renderNewUi = () => {
+    return (
+      <div className="section-tables-stylized">
+        <h2 className="title-page">
+          Push Transaction <InfoIcon style={{ marginLeft: '5px', width: '14px', height: '14px' }} />
+        </h2>
+        <TxTextInput
+          ref={textRef}
+          buttonClicked={buttonClicked}
+          action="Push Transaction"
+          onChange={handleChangeData}
+          otherAction="decode"
+          link="/decode-tx/"
+          newUiEnabled
+          placeholder="E.g.: XXXXXXXX"
+          helpText={
+            <div className="pushtx-helptext">
+              <span>
+                Write your transaction in hex value and click the button to send it to the network.
+              </span>
+              <div className="pushtx-note">
+                <div
+                  style={{
+                    width: '12px',
+                    height: '12px',
+                    display: 'flex',
+                    justifyContent: 'center',
+                    alignItems: 'center',
+                  }}
+                >
+                  <InfoIcon width={12} height={12} />
+                </div>
+
+                <span> We do not push blocks to the network, only transactions</span>
+              </div>
+            </div>
+          }
+        />
+        {canForce ? renderForceCheckbox() : null}
+        {success ? (
+          <span className="text-success">Transaction pushed to the network with success!</span>
+        ) : null}
+        <div
+          style={{
+            display: 'flex',
+            justifyContent: 'center',
+            alignItems: 'center',
+            width: '100%',
+            marginTop: '20px',
+          }}
+        >
+          <NewHathorAlert
+            type="error"
+            text="Could not decode this data to a transaction"
+            ref={alertNotFound}
+          />
```

Comment:
issue: This is not the only error message possible. For example, try to copy the raw data from an existing transaction and input it on this screen to send it again. On the old ui the message is `Transaction already exists {tx_id}`.

The way this is implemented, no other messages will be shown, even if necessary.